index.js

```diff
@@ -96,6 +96,8 @@ function toRunCommand (inspectObj, name) {
     })
   }
 
+  if (hostcfg.ExtraHosts) rc = appendJoinedArray(rc, '--add-host', hostcfg.ExtraHosts, ' ')
```

Comment:
No need for the `if` check either, since `appendArray()` will just skip if the array isn't valid.